src/runtime_src/doc/toc/xrt_native_apis.rst

```diff
@@ -0,0 +1,618 @@
+.. _xrt_native_apis.rst:
+
+XRT Native APIs
+===============
+
+From 2020.2 release XRT provides a new XRT API set in C, C++, and Python flavor. This document introduces the usability of C and C++ APIs.
+
+To use the native XRT APIs, the host application must link with the **xrt_coreutil** library.
+
+.. code-block:: shell
+
+    g++ -g -std=c++14 -I$XILINX_XRT/include -L$XILINX_XRT/lib -o host.exe host.cpp -lxrt_coreutil -pthread
```

Comment:
I guess this is just a compilation example.
Can we use another compiler?
Can we use another version of C++?

Reply:
yes, just example, changed the text to "Example of g++ command"